tools/release/lp_update_recipe.py

```diff
@@ -38,6 +38,9 @@ def get_build_path(recipe_name: str) -> str:
     if "provider" in recipe_name:
         component_name = "-".join(package[2:])  # 0-1 are checkbox provider
         return f"providers/{component_name}"
+    elif "contrib" in recipe_name:
+        component_name = "-".join(package[2:])  # 0-1 are checkbox provider
+        return f"contrib/{component_name}"
```

Comment:
1 warning and 1 request:
- [checkbox-contrib-pc-sanity-edge ](https://code.launchpad.net/~checkbox-dev/+recipe/checkbox-contrib-pc-sanity-edge)recipe that you are trying to update does not exist! you must create it 
- I like the fact that you are using and updating this script but please, update the tests and the comment!